src/excmds.ts

```diff
@@ -1634,7 +1634,20 @@ export function snow_mouse_mode() {
 export function pied_piper_mouse_mode() {
     toys.music()
 }
-
+/**
+ * Drawable variant of [[no_mouse_mode]]
```

Comment:
I think this needs a little more explanation - it's not clear that this means "you can click the mouse to draw"

Reply:
We should also mention toggle_pen in this help